ooler_mqtt_bridge

```diff
@@ -137,6 +106,77 @@ async def control_power(mqtt: Client, myooler: ooler.Ooler) -> None:
             await send_update(mqtt, myooler)
 
 
+async def get_discovery_payloads(myooler: ooler.Ooler) -> Dict[str, Dict[str, str]]:
+    """Return the discovery payloads for the device"""
+    cfg_payloads = {
+        f"{config['homeassistant_prefix']}/climate/{sanitise_mac(myooler.address)}/config": {
+            "name": await myooler.get_name(),
+            "mode_state_topic": f"ooler/{sanitise_mac(myooler.address)}/state",
+            "mode_state_template": "{{ value_json.power }}",
+            "current_temperature_topic": f"ooler/{sanitise_mac(myooler.address)}/state",
+            "current_temperature_template": "{{ value_json.current_temperature }}",
+            "temperature_state_topic": f"ooler/{sanitise_mac(myooler.address)}/state",
+            "temperature_state_template": "{{ value_json.desired_temperature }}",
+            "fan_mode_state_topic": f"ooler/{sanitise_mac(myooler.address)}/state",
+            "fan_mode_state_template": "{{ value_json.fan_mode }}",
+            "fan_mode_command_topic": f"ooler/{sanitise_mac(myooler.address)}/fan/set",
+            "mode_command_topic": f"ooler/{sanitise_mac(myooler.address)}/power/set",
+            "temperature_command_topic": f"ooler/{sanitise_mac(myooler.address)}/temperature/set",
+            "modes": ["auto", "off"],
+            "temperature_unit": "C",
+            "temp_step": 1,
+            "unique_id": myooler.address,
+            "device": device_definition,
```

Comment:
This function currently fails due to `device_definition` not being in scope here

Reply:
Dang. Another merge-tastrophie 🤦.
This explains why my Ooler wasn't responding last night!